scripts/storjsimscript.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+clear
+echo "==========================="
+echo "Tool for quick storj test "
+echo "==========================="
+UPLOADFILE=""
+DOWNLOADFILE=""
+
+function installStorjRelease {
+    shutdown
+    cd $HOME
+    echo "Current Working Directory -->" $PWD 
+    if [ -d "storj" ]
+    then
+        echo "directory exists, deleting....."
+        #rm -rf storj
+    else
+        echo "directory doesn't exists"
+    fi
+
+    read -p 'Enter the Release :' storjRelease
+    echo "Downloading $storjRelease ...."
+    #git clone https://github.com/storj/storj -b $storjRelease
+    cd $HOME/storj
+    echo "build & install storj ...."
+    go install -v ./...
+}
+
+function runStorjSim {
+    echo "Running storj-sim...."
+    osascript -e 'tell app "Terminal" to do script "storj-sim network run"'
+    sleep 2
+}
+
+function uplinkUpload() {
```

Comment:
I'm thinking if the goal is to compare upload/download speeds between releases, we should have a standard size we use here for comparison and then output the difference in time

Reply:
For the goal of making sure the new releases uploads can download the old release's uploads, we should automate that with junk.

Reply:
upload:

1) make random bytes file
2) using previous release, upload to bucket
3) using new release, download from bucket
4) compare downloaded file to uploaded file for equivalency

Reply:
based on slack conversation with @littleskunk my understanding was to very that a file uploaded using a release (say 14) should be able to be downloaded with the next release(say 15). we can for add lots of features this script to do some metric analysis. I can look into the metrics extension. First I wanted to get @littleskunk opinion as well.

Reply:
That's was what I intended to describe above. Major takeaway is we don't need the user to specify the file/bucket etc. we can use random bytes and just cleanup.

Reply:
we can also combine the download and upload into a single equivalency check